src/bundle/Resources/config/views.yml

```diff
@@ -1,5 +1,7 @@
 system:
     admin:
+        user:
+            login_template: "@EzPlatformAdminUi/Security/login.html.twig"
```

Comment:
@adamwojs  How is this related to this PR?

Reply:
I've removed bundle inheritance, so if we still would like to have custom login page for administration panel we need those config lines